CHANGELOG.md

```diff
@@ -11,6 +11,63 @@ Some experimental features can be enabled / disabled using [Feature Gates](docs/
 
 ## Unreleased
 
+## 0.9.0 - 2020-08-13
+
+### Added
+
+- Add [flow exporter] feature. [Alpha - Feature Gate: `FlowExporter`]
+   * Support sending flow records using the IPFIX protocol from each Agent ([#825](https://github.com/vmware-tanzu/antrea/pull/825) [#984](https://github.com/vmware-tanzu/antrea/pull/984), [@srikartati])
+   * Add reference cookbook to visualize exported flows using Elastic Stack ([#836](https://github.com/vmware-tanzu/antrea/pull/836), [@zyiou])
+- Support [OVS hardware offload] for Pod networking: Pods can now be assigned an SR-IOV Virtual Function. ([#786](https://github.com/vmware-tanzu/antrea/pull/786), [@moshe010])
+   * Add new CI job to validate the hardware offload functionality ([@AbdYsn])
+- Support Node MTU auto-discovery in the Antrea Agent; the user can still override this value in the Agent configuration if desired. ([#909](https://github.com/vmware-tanzu/antrea/pull/909), [@reachjainrahul])
+- Enable Antrea support for the [AKS] managed K8s service, using CNI chaining and the "networkPolicyOnly" traffic mode. ([#998](https://github.com/vmware-tanzu/antrea/pull/998), [@reachjainrahul])
+- Support for NetworkPolicy tiering (ClusterNetworkPolicy only). ([#956](https://github.com/vmware-tanzu/antrea/pull/956) [#986](https://github.com/vmware-tanzu/antrea/pull/986), [@abhiraut] [@Dyanngg])
+   * The `ClusterNetworkPolicy` Feature Gate must now be enabled for the Agent (in addition to the Controller) to activate support
```

Comment:
"to activate the support" or "to activate the feature"?

Reply:
to activate support is correct, but I used "to activate this feature" for clarity